### PR TITLE
[298] Import root namespace

### PR DESCRIPTION
**Pull request by GuillaumeEscande**
Correct import code to manage the root empty namespace

### CODE REVIEW

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ElementImpl.java [hunk 1]
```diff
@@ -530,44 +530,42 @@ else if (eNotificationRequired())
...
+     * textual concrete syntax for qualified names (including use of unrestricted name notation and escaped characters, as
+     * necessary). The qualifiedName is null if this Element has no owningNamespace or if there is not a complete
+     * ownership chain of named Namespaces from a root Namespace to this Element.
      * <!-- end-user-doc -->
      * @generated NOT
      */
     @Override
     public String getQualifiedName() {
-        boolean qualifiedNameContainsNull = false;
+        String selfName = this.getName();
```
Comment (AxelRICHARD) [inline backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ElementImpl.java:542]: It seems you add the same modifications in this file and in your other PR https://github.com/eclipse-syson/syson/pull/299/. Please remove these modifications, they don't need to be in two different PRs.
Please check every other files in this PR.

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/Element.java [hunk 2]
```diff
@@ -294,6 +294,10 @@ public interface Element extends EObject {
     /**
      * Returns the value of the '<em><b>Qualified Name</b></em>' attribute.
      * <!-- begin-user-doc -->
+     * The full ownership-qualified name of this Element, represented in a form that is valid according to the KerML
```
Comment (AxelRICHARD) [inline backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/Element.java:297]: Please remove this comment.

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java [hunk 3]
```diff
@@ -257,9 +259,9 @@ public Membership resolve(String qualifiedName) {
         if (qualification == null) {
             result = resolveLocal(name);
         } else {
-            Membership namespaceMembership = resolve(qualification);
-            if (namespaceMembership != null && (namespaceMembership instanceof Namespace)) {
-                result = ((Namespace) namespaceMembership).resolveVisible(name);
+            Element namespace = resolve(qualification).getMemberElement();
+            if (namespace != null && (namespace instanceof Namespace)) {
```
Comment (AxelRICHARD) [inline backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java:263]: Useless `namespace != null`
Useless parenthesis around `namespace instanceof Namespace`.
Please prefer something like:
```
Element memberElement = resolve(qualification).getMemberElement();
if (memberElement instanceof Namespace namespace) {
   result = namespace.resolveVisible(name);
}
```

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/Element.java (hunk 2 above)
Comment (GuillaumeEscande) [reply backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/Element.java]: Done

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java (hunk 3 above)
Comment (GuillaumeEscande) [reply backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java]: Done
